### PR TITLE
KFSTP-2000 remove unused directories

### PR DESCRIPTION
**Pull request by kkronenb**

### CODE REVIEW

**Comment by kellerj** [comment]
So, who was the Keats fan? :-)

File: src/main/config/work/staging/sys/batchContainer/placeholder.txt [hunk 1]
```diff
@@ -1,105 +0,0 @@
...
-  Adieu! the fancy cannot cheat so well
-    As she is fam'd to do, deceiving elf.
-  Adieu! adieu! thy plaintive anthem fades
-    Past the near meadows, over the still stream,
-      Up the hill-side; and now 'tis buried deep
-          In the next valley-glades:
-    Was it a vision, or a waking dream?
-      Fled is that music:--Do I wake or sleep?                 80
-
--- John Keats
```
Comment (bhutchinson) [inline src/main/config/work/staging/sys/batchContainer/placeholder.txt]: sad to see this one go

File: pom.xml [hunk 2]
```diff
@@ -294,41 +294,9 @@
...
-            <artifactId>maven-resources-plugin</artifactId>
-            <executions>
-              <execution>
-                <id>copy-resources</id>
-                <phase>package</phase>
-                <goals>
-                  <goal>copy-resources</goal>
-                </goals>
-                <configuration>
-                  <outputDirectory>${work.base.dir}</outputDirectory>
```
Comment (bhutchinson) [inline pom.xml]: should probably delete the work.base.dir property too

Comment (kkronenb) [inline pom.xml]: done

**Pull request merged**